docs/report_queries/missing_subset_declaration.md

```diff
@@ -0,0 +1,17 @@
+# Missing Subset Declaration
+
+**Problem:** A subset is used in an annotation, but is not properly declared as a child of oboInOwl:SubsetProperty. This can cause problems with conversions to OBO format.
+
+**Solution:** Make the subset a child of oboInOwl:SubsetProperty.
+
+```sparql
```

Comment:
This query currently returns a line in the report each time the subset is used as an object to `oboInOwl:inSubset`. That greatly increases the number of errors when really, this should just be one error.

If we rewrite the query slightly, we can get just one error per subset:

```
SELECT DISTINCT ?entity ?property ?value WHERE {
    VALUES ?property { rdfs:subPropertyOf }
    ?x oboInOwl:inSubset ?entity .
    FILTER NOT EXISTS { ?entity ?property oboInOwl:SubsetProperty }
}